pkg/machines/actions.es6

```diff
@@ -115,6 +123,14 @@ export function enableLibvirt(enable, serviceName) {
     return virt('ENABLE_LIBVIRT', { enable, serviceName });
 }
 
+export function volumeCreateAndAttach({ connectionName, poolName, volumeName, size, format, target, permanent, hotplug, vmName }) {
+    return virt('VOLUME_CREATE_AND_ATTACH', { connectionName, poolName, volumeName, size, format, target, permanent, hotplug, vmName });
```

Comment:
I 'd rather have a consistent naming for action names.
ex:
VOLUME_CREATE_AND_ATTACH vs ATTACH_DISK.
There is NOUN_VERB vs VERB_NOUN.
I would stick to some pattern to allow better filtering and grouping of actions. But I can fix it in another PR.

Reply:
done